lib/matplotlib/legend.py

```diff
@@ -616,11 +616,12 @@ def _init_legend_box(self, handles, labels, markerfirst=True):
         for orig_handle, lab in zip(handles, labels):
             handler = self.get_legend_handler(legend_handler_map, orig_handle)
             if handler is None:
+                fragment = "#creating-artists-specifically-for-adding-to-the-legend-aka-proxy-artists"
```

Comment:
you can also use `\` to split a line

That is a bit of an absurd ref target...